distribution/lib/Standard/AWS/0.0.0-dev/src/Database/Redshift/Internal/Redshift_Dialect.enso

```diff
@@ -1,9 +1,10 @@
-from Standard.Base import all
+from Standard.Base import all hiding Errors
 
 import Standard.Table.Internal.Naming_Helpers.Naming_Helpers
 from Standard.Table import Aggregate_Column
 from Standard.Table import Value_Type
 
+from Standard.Database import Errors
```

Comment:
This [change](a30ecfbf47539c17d3660efe460452aff2e85096) is slightly unfortunate. It forces `Standard.Database.Main` to be parsed into `IR` - as without `IR` for the `Main` module we currently refuse to do any imports from the library.

Reply:
On a second thought, the `Errors` import is needed otherwise we get:
```
Standard/AWS/0.0.0-dev/src/Database/Redshift/Internal/Redshift_Dialect.enso:141:61: error: The name `Errors.Unsupported_Database_Operation` could not be found.
  141 |     check_aggregate_support : Aggregate_Column -> Boolean ! Errors.Unsupported_Database_Operation
      |                                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```